cmd/shell.go

```diff
@@ -61,8 +61,13 @@ type Config struct {
 	}
 
 	WFE struct {
-		BaseURL       string
-		ListenAddress string
+		BaseURL                     string
+		ListenAddress               string
+		AccountKeyRequestsPerSecond int
```

Comment:
Should include a default value for the rate limit config fields in boulder-config.json and boulder-test-config.json so that code gets exercised regularly.